awslogs/bin.py

```diff
@@ -99,6 +99,13 @@ def add_date_range_arguments(parser, default_start='5m'):
                             dest='watch',
                             help="Query for new log lines constantly")
 
+    get_parser.add_argument("-i",
+                            "--interval",
+                            dest='interval',
+                            type=int,
+                            default=1,
+                            help="Interval at which to query for new log lines")
```

Comment:
specifically the interval in seconds

Reply:
Another Good catch here, I updated my pull request.